addons/stock/models/stock_quant.py

```diff
@@ -646,6 +646,43 @@ def _get_quants_action(self, domain=None, extend=False):
             })
         return action
 
+    def _create_or_update_neg_inv(self, existing_conflict_invs=None):
```

Comment:
I would not create this method and put the content in _run_conflict_inventory_tasks